collectors/kvg/collector/collector.go

```diff
@@ -8,7 +8,9 @@ import (
 )
 
 type Collector interface {
-	Run(IDs []string, runRemove bool)
+	Run(IDs []string)
+	RunSingle(ID string)
+	SubjectToID(subject string) string
 	SubjectsToIDs(subjects []string) []string
```

Comment:
Should we remove subject to ids now as we could simply use `SubjectToID` on the array?